### ISSUE TITLE
NFC scanner

### ISSUE THREAD

**Issue by AndreasWebdev** [EdwynZN/amiibo_network#4]
I am not sure if this is possible through flutter, but maybe consider adding an NFC scanner. It is the only feature that is really missing for a perfect Amiibo collection app!

**Comment by EdwynZN** [comment]
Yeah that is possible in Flutter with some nfc packages available. I discarded the idea because I don't have an Android device with NFC to test them out and I thought it was a feature people wouldn't use, but if collectors really like that feature I'll see a way to add it in the future.

**Comment by AndreasWebdev** [comment]
I was writing a script for a comparison-video of the top 10 amiibo collection apps and this one seemed the best of them all with this little feature being the only thing that could really be improved on.

I wrote an amiibo detector a little while ago (though it was in Java) so if you need help or have any questions I could retrieve that old project and see how I did it

**Comment by EdwynZN** [comment]
that sounds awesome, I've been looking around how to use the NFC and I think it is possible to implement it, I think I can work with it this weekend. The amiibo detector on java could be really useful if I can connect it with the flutter engine (at least in Android) or at least to understand the Amiibo NFC tag to implement it directly in dart
